src/Http/Controllers/Controller.php

```diff
@@ -116,32 +116,44 @@ public function insertUpdateData($request, $slug, $rows, $data)
         return $data;
     }
 
-    public function validateBread($request, $data)
+    public function validateBread($request, $data, $action = null)
     {
         $rules = [];
         $messages = [];
+        $customAttributes = [];
 
         foreach ($data as $row) {
+            if ($row->display_name) $customAttributes[$row->field] = $row->display_name;
```

Comment:
Are we enforcing a coding standard? The way the if is expressed is not allowed on PSR-2. It MUST have the brackets. But that if we're enforcing that standard.

Reply:
Don't worry, StyleCI sends in PRs auto for those :)

Reply:
@samtheson is correct.  We like having StyleCI be happy when the PR is opened, but if it's not, StyleCI will automatically open a PR to fix it once the PR gets merged.